tests/functional/modules/test_zos_copy_func.py

```diff
@@ -1976,7 +1976,7 @@ def test_ensure_copy_file_does_not_change_permission_on_dest(ansible_zos_module,
 def test_copy_dest_lock(ansible_zos_module, ds_type, f_lock ):
     hosts = ansible_zos_module
     data_set_1 = get_tmp_ds_name()
-    data_set_2 = get_tmp_ds_name()
+    data_set_2 = get_tmp_ds_name(symbols=True)
```

Comment:
symbols=True has the problem of being random, so you won't know for sure which special characters you are getting I would suggest to use something like

```suggestion
    data_set_2 = get_tmp_ds_name(llq_size=4)
    data_set_2 = f"{data_set_2}$#@"
```

Will will ensure every time we run this test case we are always testing the same special characters.